### ISSUE TITLE
Give any selected media app priority

### ISSUE THREAD

**Issue by ghokun** [TheBoredTeam/boring.notch#116]
First of all thanks for this app it is amazing!

When I open multiple applications that play media as following:
<img width="314" alt="Screenshot 2024-09-22 at 20 21 07" src="https://github.com/user-attachments/assets/2ecb1ebd-94a4-4f44-8dc3-e9c8377b9903">

Notch shows the one that is started later.
<img width="481" alt="Screenshot 2024-09-22 at 20 21 15" src="https://github.com/user-attachments/assets/6488e8ec-9bfe-4cc5-a701-9cecad46a590">

Following options would solve this problem:
1. An option to select/prioritize apps so it becomes sticky and does not change.
2. Another alternative would be displaying all media in the notch.

PS: I checked existing issues and did not see a similar issue. If this is a duplicate please guide me.

**Comment by nicholascarey** [comment]
I agree with this - No. 1 specifically. I commented on another issue about option to ignore specific apps, but this is a better option IMO. Personally i only want to use it with Music.app.  

It's also quite bugged when trying to play fullscreen video in VLC/IINA - it fails to hide and shows the currently paused song in Music with an active player. I have to quit boringNotch. This would (should) solve that issue. 

So essentially - option to include apps or option to ignore apps, but i would prefer the former.

**Comment by richardkunkli** [comment]
Hi,
Thanks for the feedback. Please check my answer in #117 , maybe that should be our workaround. Dealing with hierarchical orders also not a bad idea, but still has the unreliability of macOS global media center.

**Comment by Alexander5015** [comment]
Resolved in the latest RC
